packages/components/src/components/pin-code/__snapshots__/pin-code-styles.spec.ts.snap

```diff
@@ -37,26 +49,28 @@ input:disabled {
   -webkit-text-fill-color: #949598;
 }
 @media(hover:hover) {
-  input:not(:disabled):not(:focus):not([readonly]):hover {
+  input:not(:disabled):not(:focus):not([readonly]):hover,label:hover~.wrapper input:not(:disabled):not(:focus):not([readonly]) {
```

Comment:
Is it intended that we select the `label` tag here, while the rest of the css refers to the `.label` class?

Reply:
I think that's fine, because independently of the css class we'll always have a label tag. In the functional label component we also have the description element which needs to be styled to, that's we rely on .label class there.